symfony/plugins/orangehrmLeavePlugin/Api/EmployeeLeaveRequestAPI.php

```diff
@@ -68,6 +71,20 @@ class EmployeeLeaveRequestAPI extends Endpoint implements CrudEndpoint
         self::MODEL_DETAILED => LeaveRequestDetailedModel::class,
     ];
 
+    protected ?LeaveApplicationService $leaveApplicationService = null;
+    protected ?LeaveAssignmentService $leaveAssignmentService = null;
+
+    /**
+     * @return LeaveAssignmentService
+     */
+    public function getLeaveAssignmentService(): LeaveAssignmentService
```

Comment:
make this getter protected
`protected function getLeaveAssignmentService()`

Reply:
Done